From 102839cfebc9be3dcf65aa3c2a46051bb3c850c2 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 15 Dec 2009 13:39:50 +0800
Subject: [PATCH 139/505] ENGR00119177 vpu test: blank tv before mode change

We have to blank tv fb to avoid tv restart during power on, there is
chance to make ipu hw error.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 test/mxc_vpu_test/display.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_vpu_test/display.c b/test/mxc_vpu_test/display.c
index 9dc11f6..7518551 100644
--- a/test/mxc_vpu_test/display.c
+++ b/test/mxc_vpu_test/display.c
@@ -131,6 +131,7 @@ v4l_display_open(struct decode *dec, int nframes, struct rot rotation, Rect crop
 	tv_mode = getenv("VPU_TV_MODE");
 
 	if (tv_mode) {
+		err = system("/bin/echo 1 > /sys/class/graphics/fb1/blank");
 		if (!strcmp(tv_mode, "NTSC")) {
 			err = system("/bin/echo U:720x480i-60 > /sys/class/graphics/fb1/mode");
 			out = 5;
-- 
1.8.0

